tests/concurrent_client_request_test.go

```diff
@@ -129,8 +129,10 @@ func TestConcurrentClientRequest(t *testing.T) {
 	stopCh := make(chan struct{})
 	defer close(stopCh)
 	// Run two agents
-	runAgent(proxy.agent, stopCh)
-	runAgent(proxy.agent, stopCh)
+	cs1 := runAgent(proxy.agent, stopCh)
+	cs2 := runAgent(proxy.agent, stopCh)
+	waitForHealthyClients(t, 1, cs1)
```

Comment:
Maybe use an appropriatly named variable rahter than 1 to make the code clearer.

Reply:
Renamed to `waitForConnectedServerCount`, which hopefully makes it clear what the 1 is doing? If you'd still like variables instead I can make that change.